docs/setup_development.rst

```diff
@@ -153,21 +153,25 @@ file is saved. They are made available on your host machine by forwarding the
 following ports:
 
 * Source Interface: `localhost:8080 <http://localhost:8080>`__
-* *Journalist Interface*: `localhost:8081 <http://localhost:8081>`__
+* Journalist Interface: `localhost:8081 <http://localhost:8081>`__
```

Comment:
I'm not sure why it _isn't_ that way with "Source Interface" above, but throughout all our docs we italicize any terms that are part of our glossary: https://docs.securedrop.org/en/stable/glossary.html

```suggestion
* *Journalist Interface*: `localhost:8081 <http://localhost:8081>`__
```